README.md

```diff
@@ -94,14 +96,11 @@ Read the [Feature Overview](https://kubeshop.github.io/monokle/features/) docume
 
 There is a lot of functionality currently being considered:
 
-- Creation of resources/projects - see [Resource creation/manipulation](https://github.com/kubeshop/monokle/projects/4).
-- Improved Cluster functionality - see [Cluster integration](https://github.com/kubeshop/monokle/projects/8).
-- Plugin/extension mechanism - see [Extension mechanism for plugins](https://github.com/kubeshop/monokle/issues/177).
-- Improved resource filtering/navigation - see [Resource Navigation](https://github.com/kubeshop/monokle/projects/2).
-- Improved Helm integration - see [Helm integration](https://github.com/kubeshop/monokle/projects/12).
+https://app.zenhub.com/workspaces/monokle-620a8eb717b2fa0010645831/roadmap?invite=true
```

Comment:
This requires an account on Zenhub... maybe we could point to a Google document or just add a short version of the roadmap here?

Reply:
I thought that this was public (at least it showed as such in Zenhub)